resources/openapi.yaml

```
@@ -23,18 +23,17 @@ components:
           type: string
         name:
           type: string
-        icon-url:
+        iconURL:
```

```suggestion
        iconUrl:
```

URL は camelCase では `iconUrl` 派というのを表明します🙋‍♂️
ID は `userId` 派

---

https://qiita.com/gamako/items/a5b1428c9b5c8d6f0694

---

TypeScript の コーディング規約には特にルールなさそうだな
https://typescript-jp.gitbook.io/deep-dive/styleguide

---

じゃあ一旦 Google の Java コーディング規約 (基本的に全て先頭だけ大文字) でやっていきます :roger: